.github/workflows/static-analysis.yml

```
@@ -0,0 +1,48 @@
+name: "Static Analysis"
+
+on:
+    pull_request:
+        branches:
+            - "[0-9]+.[0-9]+"
+            - "[0-9]+.x"
+    push:
+        branches:
+            - "[0-9]+.[0-9]+"
+            - "[0-9]+.x"
+
+jobs:
+    static-analysis-phpstan:
+        name: "Static Analysis with PHPStan"
+        runs-on: "ubuntu-20.04"
+        strategy:
+            matrix:
+                include:
+                    - { php-version: "8.1", database: "mariadb:10.11", dependencies: "highest", pimcore_version: "11.x-dev", phpstan_args: "", experimental: true }
```

We should test lowest and highest dependencies
```suggestion
                    - { php-version: "8.1", database: "mariadb:10.11", dependencies: "lowest", pimcore_version: "11.x-dev", phpstan_args: "", experimental: true }
                    - { php-version: "8.1", database: "mariadb:10.11", dependencies: "highest", pimcore_version: "11.x-dev", phpstan_args: "", experimental: true }
```

---

You are right! I was just testing in general, because we have to add the AdminBundle to the dependencies too, but we are working on some refactorings